tests/test_fs.py

```diff
@@ -122,3 +130,31 @@ def test_fsspec():
         data = fp.read()
     assert data == b"123", "unexpected data"
     fs.rm(filepath)
+
+
+def test_empty_directory():
+    fs = fsspec.filesystem("v3io")
+    path = f"/{test_container}/{test_dir}/test_empty_directory"
+    makedir(path)
+    try:
+        result = fs.ls(path)
+        assert result == []
+    finally:
+        fs.rm(path)
```

Comment:
After running it on my setup I saw that it did not delete the folder...


Also It seems that the method delete_object - (that we eventually reach it) is deprecated according to the comments.

Reply:
Files are deleted just fine, but I see that the empty directory is not deleted. I seem to remember that the web API might not support deleting directories. I'll check with @assaf758. Anyway, it's not related to the v3io-py method being superficially deprecated.

Reply:
Added `rmdir` to make sure that there is a slash at the end of the path.